src/pokemon_animation.c

```diff
@@ -523,6 +523,8 @@ static void Task_HandleMonAnimation(u8 taskId)
         sprite->data[2] = gTasks[taskId].tSpeciesId;
         sprite->data[1] = 0;
 
+        if (gBattleStruct->battlerKOAnimsRunning > 0)
```

Comment:
```suggestion
        // Task_HandleMonAnimation handles more than just KO animations,
        // but if the counter is non-zero then only KO animations are running.
        // This assumption is not checked.
        if (gBattleStruct->battlerKOAnimsRunning > 0)
```
Maybe something like this to document the assumption? Happy to drop or reword the last line, I just wanted to put some words to make it clear that there's nothing structurally in the code preventing the counter from being non-zero and another animation running.

Reply:
I can take it as is.
We'll deal with it if it comes to it.